### PR TITLE
Custom informers

### PR DESCRIPTION
**Pull request by zloper**
- add activity for choice lang
- add multiple chioce
- add env updater (for StringSets)

### CODE REVIEW

File: app/src/main/java/lang_import/org/app/ReaderActivity.kt [hunk 1]
```diff
@@ -17,13 +18,42 @@ import android.widget.Toast
...
-    val reader by lazy { FeedReader("https://habr.com/rss/all/", this) }
+    //val reader by lazy { FeedReader("https://habr.com/rss/all/", this) }
+    //draft BD
+    val informersMap:HashMap <String,String> = hashMapOf(
+            "HABR" to "https://habr.com/rss/all/",
+            "Yandex.science" to "https://news.yandex.ru/science.rss",
+            "mail.ru" to "https://news.mail.ru/rss/"
+    )
+    var informerURL=""
+    val reader by lazy { FeedReader(informerURL, this) }
```
Comment (reddec) [inline app/src/main/java/lang_import/org/app/ReaderActivity.kt]: A bit strange: if `reader` once invoked, then it is never initialized again even if `informerURL` changed

Comment (zloper) [reply app/src/main/java/lang_import/org/app/ReaderActivity.kt]: Yes it is. Now we can use new informer only after restart app. This code is only concept for load multiple informers. Need refactor reader or add some refresh functional. Or we can not use multiple informers

Comment (reddec) [reply app/src/main/java/lang_import/org/app/ReaderActivity.kt]: Maybe just remove lazy initialization and implement the getter?

Comment (zloper) [reply app/src/main/java/lang_import/org/app/ReaderActivity.kt]: Done :)

File: app/src/main/java/lang_import/org/app/ReaderActivity.kt [hunk 2]
```diff
@@ -18,45 +18,58 @@ import android.widget.Toast
...
             "mail.ru" to "https://news.mail.ru/rss/"
     )
-    var informerURL=""
-    val reader by lazy { FeedReader(informerURL, this) }
+    var informerURL = ""
     private lateinit var recyclerView: RecyclerView
     private lateinit var viewAdapter: RecyclerView.Adapter<*>
     private lateinit var viewManager: RecyclerView.LayoutManager
 
+    fun get_reader(cotext: Context): FeedReader {
```
Comment (reddec) [inline app/src/main/java/lang_import/org/app/ReaderActivity.kt]: Maybe better by [getter](https://kotlinlang.org/docs/reference/properties.html)?

Comment (zloper) [reply app/src/main/java/lang_import/org/app/ReaderActivity.kt]: done 2 :)

**Comment by reddec** [comment]
LGTM!

**Review by reddec — APPROVED** [review]

**Pull request merged**